tools/prepare-release.sh

```diff
@@ -0,0 +1,232 @@
+#!/usr/bin/env bash
+# -*- shell-script[bash] -*-
+# Copyright 2022 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+set -eu -o pipefail
+# set -x
+
+MY_NAME="Firecracker $(basename "$0")"
+FC_TOOLS_DIR=$(dirname $(realpath $0))
+FC_ROOT_DIR=$FC_TOOLS_DIR/..
+
+function say {
```

Comment:
Do you mean `say` and `die`? I can move those, but they are just a few lines, so it won't save much. For the rest of the functions I have copied them over but also slightly modified them with simplifications and fixes. The plan is I will remove them from `devtool` after the new release is done. I am doing it in 2 steps so that we have a back-up plan in case the new release process does not work.

Reply:
Yes, I like it a lot this way.